### ISSUE TITLE
Allow options to be passed to sendMail()

### ISSUE THREAD

**Issue by mozmorris** [bgag/keystone-nodemailer#3]
As far as I can tell, it's not possible to pass options through to sendMail.

https://github.com/bgag/keystone-nodemailer/blob/master/index.js#L74-L82

**Comment by bergos** [comment]
Yes, we could add this feature. The current code was taken from keystone with minimal changes for maximum compatibility. Do you have a proposal or even a PR?
